client/state/selectors/is-inline-support-article-visible.js

```diff
@@ -1,9 +1,7 @@
-import { get } from 'lodash';
-
-import 'calypso/state/inline-support-article/init';
-
 /**
- * @param {object} state Global app state
  * @returns {object} ...
  */
-export default ( state ) => get( state, 'inlineSupportArticle.isVisible', false );
+export default () => {
```

Comment:
It's preferred for each export to provide a name, that makes it easier to follow when debugging with the browser dev tools.